eng/common/scripts/Helpers/Metadata-Helpers.ps1

```diff
@@ -113,19 +129,20 @@ function GenerateDocsMsMetadata($language, $languageDisplayName, $serviceName, $
     $msauthor = $author
   }
   $date = Get-Date -Format "MM/dd/yyyy"
-  $header = @"
----
-title: $langTitle
-description: $langDescription
-author: $author
-ms.author: $msauthor
-ms.date: $date
-ms.topic: reference
-ms.devlang: $language
-ms.service: $msService
----
-"@
-  return $header
+
+  $metadataTable = @{
+    "title"= $langTitle
+    "description"= $langDescription
+    "author"= $author
+    "ms.author"= $msauthor
+    "ms.data"= $date
+    "ms.topic"= "reference"
+    "ms.devlang"= $language
+    "ms.service"= $msService
+  }
+  $updatedMetadata = compare-and-merge-metadata -original $originalMetadata -updated $metadataTable
+  return "---
```

Comment:
looks like you have a newline in the middle of this string which will break.

Reply:
After a second thought, I will have another PR to change this to 
"---`r`n$updatedMetadata---`r`n". It is currently working somehow.